src/Uno.UWP/Storage/StorageFolder.cs

```diff
@@ -8,14 +8,16 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Uno.Extensions;
-using Windows.Foundation;
-
+using Windows.Foundation;
+
 #if __IOS__
 using UIKit;
 using Foundation;
 #endif
-
-
+
+// small, innocent change to force build restart.

```

Comment:
Remove this please

Reply:
Of course. This line already fulfill what I want - build was restarted and succeded :)